microshift_install_get_ready/microshift-install-get-ready.adoc

```diff
@@ -16,3 +16,82 @@ include::modules/microshift-install-system-requirements.adoc[leveloffset=+1]
+
+[id="microshift-get-ready-install-rhel-types_{context}"]
+== {op-system-base} installation types
+
+Where you want to run your cluster and what your application needs to do determine the {op-system-base} installation type that you choose. For every installation target, you must configure both the operating system and {microshift-short}. Take into consideration your application storage needs, networking for cluster or application access, and your authentication and security requirements.
+
+Understand the differences between the valid {op-system-base} installation types available, along with the support scope for each and the tools used.
+
+[id="microshift-get-ready-install-rpm_{context}"]
+=== Using RPMs, or package-based installation
+
+This is simple installation type uses a basic command to install {microshift-short} on an existing {op-system-base} machine. See xref:../microshift_install_rpm/microshift-install-rpm.adoc#microshift-install-rpm[Installing from an RPM package]. No other tools are required unless you are also installing a {op-system-base} system or virtual machine (VM) at the same time.
+
+[id="microshift-get-ready-install-rhel-image-based_{context}"]
+=== {op-system-base} image-based installations
+
+Image-based installation types involve creating an `rpm-ostree`-based, immutable version of {op-system-base} that is optimized for edge deployment.
+
+* {op-system-ostree} can be deployed to the edge in production environments. This installation type can be done with network connections or offline, depending on the local environment.
+
+* Image mode for {op-system-base} is available as Technical Preview. This image-based installation type is based on OCI container images and bootable containers. See link:https://developers.redhat.com/articles/2024/09/24/bootc-getting-started-bootable-containers[bootc: Getting started with bootable containers] for an introduction to bootc technology.
+
+When choosing an image-based installation, consider whether the installation target is intended to be in an offline or networked state, where you plan to build system images, and how you plan to load your {op-system-bundle}. Use the following scenarios as general guidance:
+
+** If you build either a fully self-contained {op-system-ostree} or image mode for {op-system-base} ISO outside a disconnected environment, then install the ISO locally on your edge devices, you likely do not need an RPM repository or a mirror registry.
+** If you build an ISO outside a disconnected environment that does not include the container images, but consists of only the RPMs, you need a mirror registry inside your disconnected environment from which to pull container images.
```

Comment:
```suggestion
** If you build an ISO outside a disconnected environment that does not include the container images, but consists of only the RPMs, you need a mirror registry inside your disconnected environment. You will pull container images from the mirror registry.
```
I split this sentence in two to make it a little easier to parse. I realize that in doing so, I used the future tense ("will"), but I think it's acceptable here because this is a planning document and you're writing about things that will happen.